src/status_im/data_store/realm/schemas/account/migrations.cljs

```diff
@@ -159,77 +163,9 @@
 (defn v24 [old-realm new-realm]
   (log/debug "migrating v24 account database"))
 
-(defn v25 [old-realm new-realm]
```

Comment:
This is ok to remove in develop as well?

Reply:
yes, it is ok